HLTrigger/HLTcore/interface/TriggerExpressionData.h

```diff
@@ -224,7 +223,7 @@ class Data {
 
   // l1 values and status
   const L1GlobalTriggerReadoutRecord  * m_l1tResults;
-  const L1GtTriggerMenu               * m_l1tMenu;
+  std::unique_ptr<L1GtTriggerMenu>      m_l1tMenu;
```

Comment:
I just checked the code for ```L1GtTriggerMenu``` and the copy constructor does call ```buildGtConditionMap``` so you don't have to.